connd/qconnectionagent.h

```diff
@@ -144,7 +155,8 @@ private slots:
     void techTetheringChanged(bool b);
 
     void openConnectionDialog(const QString &type);
-    void setWifiTetheringEnabled();
+    void setWifiTethering();
+    void setBtTethering();
```

Comment:
The names hint towards setters but implementation is for enabling. Could be enableBtTethering and enableWifiTethering?